integration-tests/build.gradle

```diff
@@ -25,6 +27,23 @@ dependencies {
 
     androidTestImplementation project(':testing')
 }
-kapt {
-    correctErrorTypes = true
+
+pluginManager.withPlugin('kotlin-kapt') {
+    kapt {
```

Comment:
This should not be needed anymore. Corrsct error types was for Java generation AFAIK

Reply:
Done